index.js

```diff
@@ -96,7 +96,7 @@ if (argv.gasprice < 1 || argv.gasprice > 1000) throw "--gasprice must be between
 
   // Transaction parameters
   const transactionOptions = {
-    gas: 12000000, // 12MM is very high. Set this lower if you only have < 2 ETH or so in your wallet.
+    gas: 8000000, // Based on empirical observation of Kovan deployments.
```

Comment:
Do you think it's more maintainable to just estimate gas and multiply by 1.1 or 1.2?